src/app/docs/render.tsx

```diff
@@ -27,6 +27,11 @@ export const RenderDocs: React.FC<Props> = ({ topics, children }) => {
   const [init, setInit] = useState(false)
   const [navOpen, setNavOpen] = useState(false)
   const theme = useTheme()
+  const pathname = usePathname()
+
+  useEffect(() => {
+    if (navOpen) setNavOpen(false)
+  }, [pathname])
```

Comment:
Does this effect run when query params change? If so, it probably should not.

Reply:
I tested this and it is not affected by query params, `pathname` disregards anything after `?`